main.go

```diff
@@ -81,9 +82,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	inspect, _, err := dockerClient.ImageInspectWithRaw(context.Background(), originalImageID.String())
```

Comment:
I didn't do this later, but we should handle errors in the same way we do above.